developers/integrations/compute-infrastructure/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Compute Infrastructure
+sidebar_position: 2
+image: og/integrations/home.jpg
+---
+
+Compute Infrastructure solutions provide managed platforms for developing, deploying, and scaling computationally intensive workloads.
```

Comment:
Consider swapping these around:

Compute Infrastructure solutions provide managed platforms for computationally intensive workloads. Use these platforms to develop, deploy, and scale your application.